docs/guide/templates.md

```diff
@@ -291,12 +368,18 @@ Linking to other Locations is done with a [native `path()` Twig helper](http://
 <a href="{{ path( location ) }}">Some link to a Location</a>
 ```
 
-If you don't have the Location object, but only its ID, you can generate the URLAlias the following way:
+If you don't have the Location object, but only its Location ID, you can generate the URLAlias the following way:
```

Comment:
```suggestion
If you don't have the Location object, but only its ID, you can generate the URL alias the following way:
```
Is that Location ID or a Content ID. From the context I think it is the other but not sure, so I would left ID to be safe.